cmd/osv-scanner/main.go

```diff
@@ -174,6 +189,7 @@ func run(args []string, stdout, stderr io.Writer) int {
 					CompareLocally:        context.Bool("experimental-local-db"),
 					CompareOffline:        context.Bool("experimental-offline"),
 					ShowAllPackages:       context.Bool("experimental-all-packages"),
+					ScanLicensesSummary:   context.Bool("experimental-licenses-summary"),
 					ScanLicenses:          context.IsSet("experimental-licenses"),
```

Comment:
ScanLicenses now feels a bit redundant, maybe rename this to `CheckLicenseAllowlist`, though it feels like we can get that info just by checking if `ScanLicenseAllowlist` is empty or not. 

Though I do see the value of this making it clear the difference between no allowlist (everything's a violation) and don't check licenses (nothing is a violation).

Let's just rename it to `CheckLicenseAllowlist` for now to make it clearer.

Reply:
I think removing it is a good call. I check earlier in this file that this is being properly set so after that we can assume that if it's empty it's not set.